scaffolder-templates/roadie-agent-scaffolder-action/skeleton/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:18-alpine
 
-ARG TOKEN=example-broker-token
+ARG TOKEN=${{ values.agent_name }}
```

Comment:
This might need some validation on the template side. I'm not sure how well whitespace plays with these